src/printing.ts

```diff
@@ -497,8 +497,8 @@ class Printing {
     return this;
   }
 
-  send() {
-    return this._send(this._buffer);
+  send(timeout?: number) {
```

Comment:
I think we need to throw error here if timeout more/less the constraints from the Epson docs.
ie. (timeout < 5000 || timeout > 600000) for iOS 
throw new Error('Timeout should be an integer from 5000 to 600000*)')

Reply:
will do